### ISSUE TITLE
Set brick constants from the configuration file

### ISSUE THREAD

**Issue by lpabon** [heketi/heketi#248]
These are advanced settings only supposed to be used during development

**Comment by lpabon** [comment]
The JSON file now supports the following values:

```json
	"glusterfs" : {
			"brick_max_size_gb" : 1024,
			"brick_min_size_gb" : 1,
			"max_bricks_per_volume" : 33
		}
	}
```

**Comment by lpabon** [comment]
Download [heketi-1.0.0-45-gabbed28-issue248.linux.amd64.tar.gz](https://goo.gl/iLOZv5)

**Comment by lpabon** [comment]
@heketi/qe Please try out this file for #247 .  I think that the issue is that the `gluster start` times out.  With this version of heketi, you can add the values as mentioned above to the `glusterfs` section in the `heketi.json` file.  If the values set in the configuration file where set correctly, you should see an output similar to the following:

```
[heketi] INFO 2015/11/05 17:16:27 Loaded mock executor
[heketi] INFO 2015/11/05 17:16:28 Adv: Max bricks per volume set to 33
[heketi] INFO 2015/11/05 17:16:28 Adv: Max brick size 1024 GB
[heketi] INFO 2015/11/05 17:16:28 Adv: Min brick size 1 GB
[heketi] INFO 2015/11/05 17:16:28 Loaded simple allocator
[heketi] INFO 2015/11/05 17:16:28 GlusterFS Application Loaded
```

Please try out different values for `max_bricks_per_volume`.  My guess is that there are too many bricks being created for GlusterFS

**Comment by lpabon** [comment]
Code in https://github.com/lpabon/heketi/tree/issue248

**Comment by lpabon** [comment]
Solved by #249
